packages/ui/src/components/CardRepeater/Card.tsx

```diff
@@ -74,20 +74,20 @@ const Card = ({ index, edit, remove, error, onMove, children }: CardProps) => {
   const isFirst = index === 0;
   const isLast = index === items.length - 1;
   const lockDecrement = moveDisabledIndexes?.includes(index);
-  const isEditDisabled = disabled || editDisabledIndexes?.includes(index);
-  const isRemoveDisabled = disabled || removeDisabledIndexes?.includes(index);
+  const isEditDisabled = disabled ?? editDisabledIndexes?.includes(index);
```

Comment:
I think this should be ||.  Same with L78, L80, L81, L88, L89, L90.

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)